src/recert.rs

```diff
@@ -16,7 +22,41 @@ pub(crate) mod timing;
 pub(crate) async fn run(recert_config: &RecertConfig, cluster_crypto: &mut ClusterCryptoObjects) -> Result<RunTimes> {
     ensure_openssl_version().context("checking openssl version compatibility")?;
 
-    let in_memory_etcd_client = get_etcd_endpoint(recert_config).await?;
+    let resource_transformers = get_resource_transformers(recert_config).await?;
+    let mut in_memory_etcd_client = get_etcd_endpoint(recert_config, resource_transformers.clone()).await?;
+
+    if resource_transformers.is_some() {
```

Comment:
can we tuck this monster `if` somewhere where it doesn't impact the readability of `recert::run`? I don't want people who are trying to understand the general flow of `recert` to be intimidated by all this transofrmer/etcd-encryption stuff which is scary

Reply:
I've hidden the "storage transformers" logic under a `setup_etcd_client()` function in `recert::run`. Future readers can dig into that if interested, otherwise I think it's clear that this function does exactly what its name suggests :) 

Let me know if you were thinking of some other alternative :pray: